target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_main.c

```diff
@@ -1352,7 +1352,7 @@ static int ag71xx_rx_packets(struct ag71xx *ag, int limit)
 		dev->stats.rx_packets++;
 		dev->stats.rx_bytes += pktlen;
 
-		skb = build_skb(ring->buf[i].rx_buf, ag71xx_buffer_size(ag));
+		skb = napi_build_skb(ring->buf[i].rx_buf, ag71xx_buffer_size(ag));
```

Comment:
You need to limit this to newer kernel... check other source how to use `ifdef `based on kernel version

Reply:
I have done the change as requested.